com.visualstudio.code.appdata.xml

```diff
@@ -18,7 +18,7 @@
 		</screenshot>
 	</screenshots>
 	<releases>
-    	<release version="1.18.0" date="2017-11-08"/>
```

Comment:
Don't remove the old release (and move the latest one to the top to make appstream-compose happy).